src/components/clusters/ClusterStatus.tsx

```diff
@@ -18,10 +18,12 @@ import { CLUSTER_STATUS_LABELS } from '../../config/constants';
 
 type ClusterStatusProps = {
   status: Cluster['status'];
+  className?: string;
```

Comment:
I don't see the `className` used anywhere?

Reply:
Good catch. But it is used from the uhc-portal where a value is actually passed.

Reply:
I've been greping uhc-portal codebase for `<ClusterStatus` and couldn't see it anywhere?